src/main/java/com/alexecollins/docker/mojo/AbstractDockerMojo.java

```diff
@@ -129,6 +129,18 @@
     private String cfgPath;
 
     /**
+     * Resource to copy out of the docker container.
```

Comment:
I think these are not needed for other mojos and could be moved into `CopyMojo`? Otherwise the help for the other  mojos will be confusing.